core/src/main/java/io/aiven/klaw/service/BaseOverviewService.java

```diff
@@ -78,9 +84,139 @@ protected List<AclInfo> getAclInfoList(
     }
 
     topicOverview.setTopicInfoList(topicInfoList);
+    return topicOverview.getAclInfoList();
+  }
+
+  private List<AclInfo> sortAclInfo(List<AclInfo> aclInfo, AclGroupBy groupBy) {
+    // merge reduce
+    if (groupBy.equals(AclGroupBy.TEAM)) {
+      aclInfo = mergeReduceAclInfo(aclInfo);
+      aclInfo.sort(
+          Comparator.comparing(AclInfo::getTeamname)
+              .thenComparing(AclInfo::getEnvironmentName)
+              .thenComparing(AclInfo::getConsumergroup));
```

Comment:
probably null consumer group ? but consumergroup should always be 'na' from our recent code changes

java.lang.NullPointerException: Cannot invoke "java.lang.Comparable.compareTo(Object)" because the return value of "java.util.function.Function.apply(Object)" is null
	at java.base/java.util.Comparator.lambda$comparing$77a9974f$1(Comparator.java:473) ~[na:na]
	at java.base/java.util.Comparator.lambda$thenComparing$36697e65$1(Comparator.java:221) ~[na:na]
	at java.base/java.util.TimSort.binarySort(TimSort.java:296) ~[na:na]
	at java.base/java.util.TimSort.sort(TimSort.java:221) ~[na:na]
	at java.base/java.util.Arrays.sort(Arrays.java:1307) ~[na:na]
	at java.base/java.util.ArrayList.sort(ArrayList.java:1721) ~[na:na]
	at io.aiven.klaw.service.BaseOverviewService.sortAclInfo(BaseOverviewService.java:94) ~[classes/:na]